transforms/ebml_schema2markdown4rfc.xsl

```diff
@@ -99,6 +99,19 @@
       <xsl:value-of select="."/>
       <xsl:text>&#xa;&#xa;</xsl:text>
     </xsl:for-each>
+    <xsl:if test="ebml:implementation_note">
+      <xsl:text>implementation notes:&#xa;&#xa;</xsl:text>
+      <xsl:text>|type|note|&#xa;</xsl:text>
```

Comment:
rather than `type` I'd prefer `attribute` as that's what the implementation note refers to. (I had to go back to the EBML spec to remember what the `note_type` was for)

Reply:
resolved in https://github.com/cellar-wg/matroska-specification/pull/272/commits/0dba7599f70d3c074395c383d4fb5c061afff7f7 since I merged your change to EBML